diff --git a/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi b/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi
index 814948a6d745..a61d5a4c46c9 100644
--- a/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8173-elm.dtsi
@@ -97,10 +97,28 @@
 	trips {
 		threshold: trip-point@0 {
 			temperature = <60000>;
+			hysteresis = <2000>;
+			type = "passive";
 		};
 
 		target: trip-point@1 {
 			temperature = <65000>;
+			hysteresis = <2000>;
+			type = "passive";
+		};
+	};
+	cooling-maps {
+		map0 {
+			trip = <&target>;
+			cooling-device = <&cpu0 0 0>,
+					 <&cpu1 0 0>;
+			contribution = <3072>;
+		};
+		map1 {
+			trip = <&target>;
+			cooling-device = <&cpu2 0 0>,
+					 <&cpu3 0 0>;
+			contribution = <1024>;
 		};
 	};
 };
diff --git a/arch/arm64/boot/dts/mediatek/mt8173-oak.dtsi b/arch/arm64/boot/dts/mediatek/mt8173-oak.dtsi
index 6129c7a9264a..729845a0bd44 100644
--- a/arch/arm64/boot/dts/mediatek/mt8173-oak.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8173-oak.dtsi
@@ -57,7 +57,7 @@
 	};
 
 	chosen {
-		linux,stdout-path = &uart0;
+		stdout-path = &uart0;
 	};
 
 	firmware {
diff --git a/arch/arm64/boot/dts/mediatek/mt8173.dtsi b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
index 0b590fa62950..3db18b6bccb9 100644
--- a/arch/arm64/boot/dts/mediatek/mt8173.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8173.dtsi
@@ -272,39 +272,39 @@
 			sustainable-power = <1500>; /* milliwatts */
 
 			trips {
-				threshold: trip-point@0 {
-					temperature = <68000>;
-					hysteresis = <2000>;
-					type = "passive";
-				};
-
-				target: trip-point@1 {
-					temperature = <85000>;
-					hysteresis = <2000>;
-					type = "passive";
-				};
-
-				cpu_crit: cpu_crit@0 {
+//				threshold: trip-point0 {
+//					temperature = <68000>;
+//					hysteresis = <2000>;
+//					type = "passive";
+//				};
+
+//				target: trip-point1 {
+//					temperature = <85000>;
+//					hysteresis = <2000>;
+//					type = "passive";
+//				};
+
+				cpu_crit: cpu_crit0 {
 					temperature = <115000>;
 					hysteresis = <2000>;
 					type = "critical";
 				};
 			};
 
-			cooling-maps {
-				map@0 {
-					trip = <&target>;
-					cooling-device = <&cpu0 0 0>,
-							 <&cpu1 0 0>;
-					contribution = <3072>;
-				};
-				map@1 {
-					trip = <&target>;
-					cooling-device = <&cpu2 0 0>,
-							 <&cpu3 0 0>;
-					contribution = <1024>;
-				};
-			};
+//			cooling-maps {
+//				map0 {
+//					trip = <&target>;
+//					cooling-device = <&cpu0 0 0>,
+//							 <&cpu1 0 0>;
+//					contribution = <3072>;
+//				};
+//				map1 {
+//					trip = <&target>;
+//					cooling-device = <&cpu2 0 0>,
+//							 <&cpu3 0 0>;
+//					contribution = <1024>;
+//				};
+//			};
 		};
 	};
 
